tools/disasm/disasm.py

```diff
@@ -2,9 +2,16 @@
 
 import argparse, ast, math, os, re, struct
 import bisect
-from mips_isa import *
-from multiprocessing import *
+import multiprocessing
 from pathlib import Path
+# global mips_isa.mips_fpr_names
+import mips_isa
+
+# Consider implementing gpr naming too, but already uses abi names by default
+fpr_name_options = {
+    "numeric": mips_isa.numeric_fpr_names,
+    "o32": mips_isa.o32_fpr_names,
```

Comment:
Is it better to leave this as `o32` for clarity or change it to just `32` for consistency with objdump flags?